core/src/main/java/io/github/belgif/rest/guide/validator/core/parser/Parser.java

```diff
@@ -107,6 +103,15 @@ public <T extends Constructible> OpenApiDefinition<T> resolve(T model) {
             }
         }
 
+        private Optional<OpenApiDefinition<?>> resolveToOptional(Constructible model) {
```

Comment:
Method name: resolveReference - this is in the case where argument is a Reference
Argument can be Reference here, callers already check on it.